web/toolbar-geckoview.js

```diff
@@ -20,16 +20,24 @@
  * @property {HTMLButtonElement} download - Button to download the document.
  */
 
+const TIME_BEFORE_SHOWING_THE_BUTTON = 200;
```

Comment:
Given that the toolbar could conceivably, at some later point, contain more than just one button: 
```suggestion
const TIME_BEFORE_SHOWING_TOOLBAR = 200;
```